### PR TITLE
NIFI-5926 Added certificate commands to toolkit guide.

### PR DESCRIPTION
**Pull request by alopresto**
Thank you for submitting a contribution to Apache NiFi.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [x] Is there a JIRA ticket associated with this PR? Is it referenced 
     in the commit message?

- [x] Does your PR title start with NIFI-XXXX where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.

- [x] Has your PR been rebased against the latest commit within the target branch (typically master)?

- [x] Is your initial contribution a single, squashed commit?

### For code changes:
- [x] Have you ensured that the full suite of tests is executed via mvn -Pcontrib-check clean install at the root nifi folder?
- [ ] Have you written or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)? 
- [ ] If applicable, have you updated the LICENSE file, including the main LICENSE file under nifi-assembly?
- [ ] If applicable, have you updated the NOTICE file, including the main NOTICE file found under nifi-assembly?
- [ ] If adding new Properties, have you added .displayName in addition to .name (programmatic access) for each of the new properties?

### For documentation related changes:
- [x] Have you ensured that format looks appropriate for the output in which it is rendered?

### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

**Comment by andrewmlim** [comment]
Reviewing.

File: nifi-docs/src/main/asciidoc/toolkit-guide.adoc [hunk 1]
```diff
@@ -1111,6 +1109,9 @@ coefficient:
     7a:00:bb:07:e9:13:91:82:9b
 ```
 
+[[additional-commands]]
+==== Additional Commands
```
Comment (andrewmlim) [inline nifi-docs/src/main/asciidoc/toolkit-guide.adoc]: Change text to "Additional Certificate Commands" to be consistent with the text from the Admin Guide link.

Also, is this section within the Intermediate CA section or outside?  If outside, change to:

=== Additional Certificate Commands

File: nifi-docs/src/main/asciidoc/toolkit-guide.adoc [hunk 2]
```diff
@@ -1126,6 +1127,13 @@ coefficient:
...
+. To combine a private key in PEM format (`private.key`) and public certificate in PEM format (`certificate.pem`) into PKCS12 keystore:
+  * The following command will create the PKCS12 keystore (`keystore.p12`) from the two independent files. A Java keystore (JKS) cannot be formed directly from the PEM files:
+  ** `openssl pkcs12 -export -out keystore.p12 -inkey private.key -in certificate.pem`
+. To convert a PKCS12 keystore (`keystore.p12`) to JKS keystore (`keystore.jks`):
+  * The following command will create the JKS keystore (`keystore.jks`). The `-destalias` flag is optional, as NiFi does not currently read from a specific alias in the keystore. The user will be prompted for a keystore password, which must be set and have minimum 8 characters, and a key password, which can be the same as the keystore password or different:
+  ** `keytool -importkeystore -srckeystore keystore.p12 -srcstoretype pkcs12 -destkeystore keystore.jks
+       -deststoretype jks -destalias nifi-key`
 
 [[tls_external-signed_ca]]
 ==== Signing with Externally-signed CA Certificates
```
Comment (andrewmlim) [inline nifi-docs/src/main/asciidoc/toolkit-guide.adoc]: Pending what is done with the Additional Certificate Commands section, this section might also need to be edited to be on the same level.

**Comment by andrewmlim** [comment]
@alopresto Besides my minor suggestions/comments, looks good!

File: nifi-docs/src/main/asciidoc/toolkit-guide.adoc [hunk 3]
```diff
@@ -889,8 +889,6 @@ In some enterprise scenarios, a security/IT team may provide a signing certifica
     └── lib
 ```
 
-[[additional-commands]]
-==== Additional Commands
 The `nifi-cert.pem` and `nifi-key.key` files should be ASCII-armored (Base64-encoded ASCII) files containing the CA public certificate and private key respectively. Here are sample files of each to show the expected format:
 
 ===== nifi-cert.pem
```
[the remark this answers is not in the document]
Comment (alopresto) [reply nifi-docs/src/main/asciidoc/toolkit-guide.adoc]: Done.

File: nifi-docs/src/main/asciidoc/toolkit-guide.adoc (hunk 2 above)
Comment (alopresto) [reply nifi-docs/src/main/asciidoc/toolkit-guide.adoc]: Done.

File: nifi-docs/src/main/asciidoc/toolkit-guide.adoc (hunk 1 above)
Comment (alopresto) [reply nifi-docs/src/main/asciidoc/toolkit-guide.adoc]: Done.

**Comment by alopresto** [comment]
Thanks Drew. Made those changes. +1 so I can merge?

**Comment by andrewmlim** [comment]
LGTM.  +1.

**Pull request closed without merging**